fastdds_python/test/api/test_datawriter.py

```diff
@@ -308,26 +308,40 @@ def test_get_liveliness_lost_status(datawriter):
     assert(0 == status.total_count_change)
 
 
-def test_get_matched_subscription_data(datawriter):
+def test_get_matched_subscription_data(datawriter, reader_topic, subscriber):
```

Comment:
why don't create a pytest fixture `datareader` that could be used in future tests?